src/structures/game_session.ts

```diff
@@ -1779,11 +1863,70 @@ export default class GameSession extends Session {
         messageContext: MessageContext,
         round: GameRound,
     ): Promise<void> {
-        round.interactionMessage = await sendInfoMessage(
-            messageContext,
-            this.generateRemainingPlayersMessage(round),
-        );
+        round.interactionMessage = await sendInfoMessage(messageContext, {
+            ...this.generateRemainingPlayersMessage(round),
+            components: this.isClipMode()
+                ? [{ type: 1, components: this.generateClipButton() }]
+                : undefined,
+        });
 
         this.startHiddenUpdateTimer();
     }
+
+    private async sendClipMessage(
+        messageContext: MessageContext,
+        round: GameRound,
+    ): Promise<void> {
+        round.interactionMessage = await sendInfoMessage(messageContext, {
+            title: i18n.translate(
+                this.guildID,
+                "misc.interaction.guess.title",
+                {
+                    songOrArtist:
+                        this.guildPreference.gameOptions.guessModeType ===
+                        GuessModeType.ARTIST
+                            ? i18n.translate(this.guildID, "misc.artist")
+                            : i18n.translate(this.guildID, "misc.song"),
+                },
+            ),
+            thumbnailUrl: KmqImages.LISTENING,
+            components: [
+                {
+                    type: 1,
+                    components: this.generateClipButton(),
+                },
+            ],
+        });
+    }
+
+    private generateClipButton(): Eris.InteractionButton[] {
+        return [
+            {
+                type: 2,
+                style: 1,
```

Comment:
Can we make this a different color?

Reply:
Sure, I can change it to be a secondary button (grey)